types/jasmine/index.d.ts

```diff
@@ -18,6 +18,7 @@
 //                 kirjs <https://github.com/kirjs>
 //                 Md. Enzam Hossain <https://github.com/ienzam>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.1
```

Comment:
According to [README](https://github.com/DefinitelyTyped/DefinitelyTyped#older-versions-of-typescript-33-and-earlier) DT only supports TypeScript >=3.4. Can you explain why is this change necessary?

Reply:
npm test jasmine fails without a comment like this, and this was the suggested comment when I ran said command

If DT wants to support minimum 3.4 then they need to fix their test tooling first

Reply:
Interesting. Maybe you have outdated `dtslint` package or something? I don't get this error with:

```
$ rm -rf node_modules
$ npm i
$ npm test jasmine
```

Reply:
Yeah I figured that... I'll update and try

Reply:
That was it. Force pushed and fixed.